domain/domain.go

```diff
@@ -301,7 +301,7 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
 	phyTblIDs := make([]int64, 0, len(diffs))
 	actions := make([]uint64, 0, len(diffs))
 	for _, diff := range diffs {
-		IDs, err := builder.ApplyDiff(m, diff)
+		IDs, err := builder.ApplyDiff(store, m, diff)
```

Comment:
Is this really needed, if already using the `Domain.store` and it is already given when `infoschema.NewBuilder(do.Store(),...` is called?